ci/docker/conda-cpp.dockerfile

```
@@ -36,6 +36,7 @@ RUN mamba install -q -y \
         compilers \
         doxygen \
         python=${python} \
+        ucx-proc=*=cpu ucx libnuma \
```

How about putting one package per line like others?

```suggestion
        libnuma \
        python=${python} \
        ucx \
        ucx-proc=*=cpu \
```